### PR TITLE
fixes for crawler compatibility

### PR DESCRIPTION
**Pull request by DMsGuild201**

### CODE REVIEW

File: collection/Daniel Nnorth; Slivers and Magical Items.json [hunk 1]
```diff
@@ -1744,7 +1744,7 @@
 			"source": "thv",
 			"name": "Clot Sliver",
 			"size": "M",
-			"type": "monstrosity (shapechanger",
+			"type": "monstrosity (shapechanger)",
```
Comment (TheGiddyLimit) [inline collection/Daniel Nnorth; Slivers and Magical Items.json]: Should be
```json
"type": {
	"type": "monstrosity",
	"tags": [
		"shapechanger"
	]
}
```

File: collection/MCDM Productions; Strongholds and Followers.json [hunk 2]
```diff
@@ -26752,7 +26752,7 @@
 							"entries": [
 								"Those who stand closest to the light cast the longest shadows."
 							],
-							"by": "Kālidāsa Dhar, Vizier to the Court of Virmastha",
+							"by": "Kalidasa Dhar, Vizier to the Court of Virmastha",
```
Comment (TheGiddyLimit) [inline collection/MCDM Productions; Strongholds and Followers.json]: The original was correct; please revert this

File: collection/Daniel Nnorth; Slivers and Magical Items.json (hunk 1 above)
Comment (DMsGuild201) [reply collection/Daniel Nnorth; Slivers and Magical Items.json]: will pull and fix

File: collection/MCDM Productions; Strongholds and Followers.json (hunk 2 above)
Comment (DMsGuild201) [reply collection/MCDM Productions; Strongholds and Followers.json]: Won't work with his Parser apparently. I'll let him know though

**Pull request closed without merging**